src/orquestra/sdk/_shared/abc.py

```diff
@@ -15,8 +15,8 @@
 
 from orquestra.sdk._shared.exceptions import WorkspacesNotSupportedError
 
-from ._logs._interfaces import LogOutput, LogReader, WorkflowLogs
 from ._spaces._structs import Project, ProjectRef, Workspace
+from .logs._interfaces import LogOutput, LogReader, WorkflowLogs
```

Comment:
**nitpick**: unless we have an `__all__` attribute in this module (`_shared.abc`), this line will re-export `LogOutput, LogReader, WorkflowLogs` under the public module `_shared.abc`, which might not be the intention. Unfortunately the same applies to all the public modules under `_shared.schema` which we might wanna keep out of scope of this ticket.